spec/unit/form_builder_spec.rb

```diff
@@ -277,6 +277,69 @@ def build_form(options = {}, form_object = Post.new, &block)
 
   end
 
+
+  context "with a button to add a new has_one relationship" do 
+    let :body do
+      build_form do |f|
+        f.inputs do
+          f.input :title
+          f.input :body
+        end
+        f.form_builder.instance_eval do
+          @object.author = User.new
+        end
+        f.inputs name: 'Author', for: :author do |author|
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.